### DISCUSSION TITLE
Automatically publish action to marketplace on release

### DISCUSSION THREAD

**Discussion by jdbruijn** [community/community#26410] [Actions]
<p>Hi,</p>
<p>I have an action, but currently have to manually publish every new release to the marketplace. I have an automatic release setup, but have to do the following steps manually to get the new version in the marketplace.</p>
<ol>
<li>Go to the repo’s releases page and click on edit on the release I want to publish to the marketplace.</li>
<li>Click on update release. Note that I do not change anything in the release and that the “Publish this Action to the GitHub Marketplace” is already checked and all checks are green.</li>
</ol>
<p>The <a href="https://help.github.com/en/actions/building-actions/publishing-actions-in-github-marketplace#publishing-an-action" rel="nofollow noopener">docs</a> of publishing actions are not very explicit about it. I’ve also found <a href="https://github.community/t/publish-to-marketplace-checkbox-no-longer-appears-cant-update-action-with-new-release/17906/2">this</a> comment from <a class="mention" href="/yanjingzhu">@yanjingzhu</a> that says it the lates version in the marketplace should update automatically.</p>
<p><a href="https://github.com/vidavidorra/github-action-renovate/releases/tag/v1.0.4" rel="nofollow noopener">This</a> is my latest release (v1.0.4), for which I have not yet done the above steps to publish it in the Marketplace. And also the <a href="https://github.com/marketplace/actions/self-hosted-renovate" rel="nofollow noopener">marketplace link</a> for reference, which shows v1.0.3 as latest release.</p>

**Comment by mscoutermarsh** [comment]


**Reply by hongaar** [reply]
@mscoutermarsh is it be possible to automatically publish new releases to the GitHub marketplace with the REST/GraphQL API so we can automate this?

**Reply by mscoutermarsh** [reply]
👋 I don't work for GitHub anymore. So I have no idea 😀

**Reply by hongaar** [reply]
Should have checked since 2+ years ago, thanks for the reply anyway!

**Comment by jdbruijn** [comment]
<p>Okay, good to know! It would be really helpfull if that can be explicitly added to the docs section I referenced in my first message. That way other people don’t have to try and search a while to figure this out.</p>

**Comment by litetex** [comment]
<p><a class="mention" href="/mscoutermarsh">@mscoutermarsh</a><br>
Any progress on this?<br>
When can the new features be expected?</p>

**Comment by tpluscode** [comment]
I just found this discussion. Has anything changed in that matter?

**Reply by mrienstra** [reply]
Started https://github.com/orgs/community/discussions/52135 in an attempt to restart this conversation, worried that this one has fallen "off radar" since it was marked as "answered" in 2020. Which is not to say it shouldn't have been marked as such at the time!

**Comment by abhishek-ws** [comment]
> Sorry! No updates at the moment. We aren’t working on it yet. Teams are focused on a lot of other features.

Hey @mscoutermarsh , Is there any update at this time ?

**Reply by mscoutermarsh** [reply]
I don't work for GitHub anymore.
